CleverPush/CleverPush/Source/CleverPush.m

```diff
@@ -351,6 +351,12 @@ + (void)getSubscriptionId:(void(^)(NSString *))callback {
     }];
 }
 
++ (void)getDeviceToken:(void(^)(NSString *))callback {
+    [self.CPSharedInstance getDeviceToken:^(NSString *callbackInner) {
```

Comment:
I think we can directly pass the callback like:
`[self.CPSharedInstance getDeviceToken:callback];`